app/javascript/components/subject/CaseStatus.js

```diff
@@ -150,15 +169,29 @@ class CaseStatus extends React.Component {
               continue exposure monitoring.
             </p>
             {this.props.has_group_members && (
-              <Form.Group className="mt-2">
-                <Form.Check
-                  type="switch"
-                  id="apply_to_group"
-                  label="Apply this change to the entire household that this monitoree is responsible for"
-                  onChange={this.handleChange}
-                  checked={this.state.apply_to_group}
-                />
-              </Form.Group>
+              <React.Fragment>
```

Comment:
This is the same code snippet as above and below. I'd pull it out into it's own method to avoid the duplication.

Reply:
the problem is pretty much the whole modal is the same in all cases except in a few spots.  I broke this out into a method but maybe should consider refactoring the whole thing at a later date

Reply:
Yeah it looks that way. I'll make a ticket to do so.